.github/workflows/updatecli.d/values.d/ironbank.yml

```diff
@@ -0,0 +1,7 @@
+config:
+  - path: dev-tools/packaging/templates/ironbank/auditbeat
+    packages: dev-tools/packaging/packages.yml
```

Comment:
I think this should not be within a path, but a different data structure... I'm thinking about this